components/builder-worker/src/runner/util.rs

```diff
@@ -202,14 +186,14 @@ pub fn docker_exporter_spec(workspace: &Workspace) -> DockerExporterSpec {
     // above. As a result, Any panics that occur are most likely due to programmer error and not
     // input validation.
 
-    let creds: JsonValue = serde_json::from_str(
-        workspace
-            .job
-            .get_integrations()
-            .first()
-            .expect("Origin integrations must not be empty")
-            .get_body(),
-    ).expect("Origin integrations body must be JSON");
+    let origin_integration = workspace.job.get_integrations().first().expect(
+        "Origin integrations must not be empty",
+    );
+
+    let creds: JsonValue = serde_json::from_str(origin_integration.get_body()).expect(
```

Comment:
Might as well remove the expect here as well, and explicitly error check (avoid potential future panic)